client/state/data-layer/wpcom/sites/plugins/new/index.js

```diff
@@ -21,6 +28,38 @@ export const uploadPlugin = ( { dispatch }, action ) => {
 	}, action ) );
 };
 
+const showSuccessNotice = ( dispatch, { name } ) => {
+	dispatch( successNotice(
+		translate( 'Successfully uploaded plugin %(name)s.', {
```

Comment:
I can see there's a dot at the end of the success notice in the code, but it's not in the screenshot. Perhaps you put the dot after you made the screenshot? :)